lib/src/storyboard_controller.dart

```diff
@@ -82,11 +83,21 @@ class StoryBoardController {
 
   bool spotLightVisible = true;
   bool showFlowInPullRequest = false;
+  static bool isStoryBoard = false;
+  static bool flitWeb = true;
 
   void attach(StoryBoardState storyBoardState) {
     this.view = storyBoardState;
   }
 
+  static bool myWebFoundation() {
+    bool isFlitWeb = kIsWeb;
+    if (StoryBoardController.isStoryBoard) {
+      isFlitWeb = flitWeb;
```

Comment:
tsy ilaina atao variable flitWeb satria constant sady in1 iany no miasa

Reply:
mitovy fona ny valeur anito une fois milance ilay application. au lieu de miantso function in b d b de antsoina in 1 ftsn de stocker ilay valiny de ilay valiny no ampesaina